tests/jax/components/training/step_test.py

```diff
@@ -276,6 +276,14 @@ def test_step(mock_trainer: Trainer) -> None:
 
     mapg_with_trust_region_step.on_training_step_fn(trainer=mock_trainer)
     old_key = mock_trainer.store.base_key
+
+    # Step with policy states
+    states = jnp.zeros((1, 5))
+    policy_states = {"agent_0": states, "agent_1": states, "agent_2": states}
+    dummy_sample.data.extras["policy_states"] = policy_states
+    metrics = mock_trainer.store.step_fn(dummy_sample)
+
+    # Step without policy states
     metrics = mock_trainer.store.step_fn(dummy_sample)
```

Comment:
both of these steps use the same dummy_sample so both have policy_states

Reply:
Ah yes. Good catch.I fixed it :+1: